HelpSource/Classes/Dictionary.schelp

```diff
@@ -10,6 +10,14 @@ Two keys match if they are strong::equal::. (i.e. == returns true.)
 The contents of a Dictionary are strong::unordered::. You must not depend on the order of items in a Dictionary.
 You must only rely on equality for the keys (e.g. symbols are ok, strings not). For identity matching see: link::Classes/IdentityDictionary::.
 
+Warning: code::nil:: can't be used as a Dictionary value.
+
+code::
+d = Dictionary();
+d.put(\a, nil); // won't work
+d.keys; // Set[]
+::
+
```

Comment:
Then your example shouldn't say "won't work" (it "works", if there is a value under `\a`). Maybe better demonstrate how it works?

code::
d = Dictionary();
d.put(\a, 440); 
d.keys; // Set[\a]
d.put(\a, nil); // removes the value 440
d.keys; // Set[]
::

Reply:
i think @telephon 's example above would be good. as @eleses pointed out, the example by @jamshark70 isn't relevant because IdentityDictionary doesn't have proto/parent. if you'd like, you can add this same documentation section to IdentityDictionary's help file and explain the more complex semantics in another PR.